### PR TITLE
Bug 1210680 - Use dialog service to show wifi bad credential dialog, …

### PR DESCRIPTION
**Pull request by raylin**
…r=gasolin

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f7359a0d2ee407651d923a3274906eb967ab038d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=50149d4a18389bcff825b60e3e92b6310c4cd27c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=837edf2332d58b44e71af75bb3132f6c88df885f

File: apps/settings/js/panels/wifi/panel.js [hunk 1]
```diff
@@ -261,36 +261,21 @@ define(function(require) {
...
+        DialogService.confirm({
+          id: 'wifi-bad-credentials-confirm',
+          args: { ssid : network.ssid }
+        }, {
+          title: 'wifi-bad-credentials-title',
+          submitButton: { id: 'ok', style: 'recommend' },
+          cancelButton: 'cancel',
+        }).then((result) => {
+          if (result.type === 'submit') {
+            self._networkList().then((networkList) => {
```
Comment (gasolin) [inline apps/settings/js/panels/wifi/panel.js]: May we eliminate the usage of 'self' here?

Comment (raylin) [inline apps/settings/js/panels/wifi/panel.js]: no problem, I'll change to .bind(this)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5a5e1eac5bfebcd4ab69fcf80a28048c7517f3d4

**Pull request merged**